packages/mask/src/plugins/Referral/helpers.tsx

```diff
@@ -0,0 +1,119 @@
+import { padStart } from 'lodash-unified'
+import { BigNumber } from '@ethersproject/bignumber'
+import type { PrefixedHexString } from 'ethereumjs-util'
+import type { ChainId } from '@masknet/web3-shared-evm'
+import { createTypedMessageMetadataReader } from '@masknet/typed-message'
+
+import { META_KEY, supportedChainId } from './constants'
+import type { ReferralMetaData, ChainAddress, ChainAddressProps, EvmAddress, Bytes32, Bytes24 } from './types'
+import schema from './schema.json'
+
+// Fast convert internal types to Buffer
+// Ensure we only convert values if we need them
+export function buf(b: Buffer | Uint8Array | PrefixedHexString | BigNumber | number | bigint): Buffer | undefined {
```

Comment:
Please use `Uint8Array` other than `Buffer`.

Reply:
@Jack-Works, I see that "buffer": "^6.0.3" is a dependency in the [packages](https://github.com/DimensionDev/Maskbook/blob/6f1f1de8c57273cdcb38858e00d94d18b6a0e023/packages/mask/package.json#L76).
Should I rewrite all functions to Uint8Array?